drivers/interrupt_controller/CMakeLists.txt

```diff
@@ -26,3 +26,9 @@ zephyr_library_sources_ifdef(CONFIG_INTC_ESP32              intc_esp32.c)
 zephyr_library_sources_ifdef(CONFIG_INTC_ESP32C3            intc_esp32c3.c)
 zephyr_library_sources_ifdef(CONFIG_SWERV_PIC               intc_swerv_pic.c)
 zephyr_library_sources_ifdef(CONFIG_VEXRISCV_LITEX_IRQ      intc_vexriscv_litex.c)
+
+if(CONFIG_INTEL_VTD_ICTL)
+	zephyr_library_include_directories(
+		 ${ZEPHYR_BASE}/arch/x86/include
+	)
+endif()
```

Comment:
The whitespace here doesn't look correct, could you compare it to zephyr/CMakeLists.txt.